src/App.jsx

```diff
@@ -17,37 +19,52 @@ function App() {
             <a className="navbar-brand">conduit</a>
             <ul className="nav navbar-nav pull-xs-right">
               <li className="nav-item">
-                <a className="nav-link">Home</a>
-              </li>
-              {/* Start logged in */}
-              <li className="nav-item">
-                <a className="nav-link">
-                  <i className="ion-compose" />
-                  &nbsp;New Post
-                </a>
-              </li>
-              <li className="nav-item">
-                <a className="nav-link">
-                  <i className="ion-gear-a" />
-                  &nbsp;Settings
-                </a>
-              </li>
-              <li className="nav-item">
-                <a className="nav-link">
-                  {/* Auth user avatar */}
-                  <img src="#" />
-                  {/* Auth user username */}
-                </a>
+                <NavLink to="/" activeClassName="active" className="nav-link">
+                  Home
+                </NavLink>
               </li>
+              {isAuth && (
+                <React.Fragment>
+                  <li className="nav-item">
+                    <a className="nav-link">
+                      <i className="ion-compose" />
+                      &nbsp;New Post
+                    </a>
+                  </li>
+                  <li className="nav-item">
+                    <a className="nav-link">
+                      <i className="ion-gear-a" />
+                      &nbsp;Settings
+                    </a>
+                  </li>
+                  <li className="nav-item">
+                    <a className="nav-link">
+                      {/* Auth user avatar */}
```

Comment:
Let's get the `authUser` from the `useAuth` hook and use it here to display the avatar and the username next to it.